packages/terra-consumer-site/webpack.config.js

```diff
@@ -15,7 +15,7 @@ const rtl = require('postcss-rtl');
 module.exports = {
   entry: {
     'babel-polyfill': 'babel-polyfill',
-    'terra-core': path.resolve(path.join(__dirname, 'src', 'Index')),
+    'terra-consumer': path.resolve(path.join(__dirname, 'src', 'Index')),
```

Comment:
is this just something we missed in our initial setup?

Reply:
i think so